tests/test_wsman.py

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env python
+# -*- mode: python -*-
```

Comment:
Your emacs isn't picking up the py extension? Adding the mode line here seems like it shouldn't be needed.